blivet/populator/helpers/lvm.py

```diff
@@ -37,6 +37,9 @@
 from .devicepopulator import DevicePopulator
 from .formatpopulator import FormatPopulator
 
+from ...staticdata.lvm_info import lvs_info, pvs_info
+
+
```

Comment:
Are you sure? My bet is it was down below, not here. We have a lot of imports like this in many files and two blank lines are right below the ``log = logging.getLogger("blivet")`` line, not before ``import logging``.